getting-started/nuget-sso.md

```diff
@@ -0,0 +1,76 @@
+
+>tip NuGet keys are helpful not only for Telerik users with Google Auth / SSO accounts but also for CI users and desktop developers. Learn more about the idea of NuGet keys and how to use them through [this excellent blog post by Lance MacCarthy](https://www.telerik.com/blogs/announcing-nuget-keys).
+
+### Generate NuGet Key
+
+Generating a key for the Telerik NuGet server (https://nuget.telerik.com/v3/index.json) is fast and easy:
+
+- Open the [NuGet keys management page](https://www.telerik.com/account/downloads/nuget-keys). Alternatively, navigate to that page through **Account Overview** > **Downloads** > **Manage NuGet Keys**
+
+- Click the **Generate Key** button to create a new NuGet key. 
+
+- Copy and then store the value before replacing the contents of your clipboard. Note that this is the only time that you will see the key. Each generated key is valid for two years (or until explicitly deleted).
+
+![Generate new key](./images/nuget-keys-telerik-001.png)
+
+
+### Using the NuGet Key for Authentication
+
+Instead of entering a username and password through a Visual Studio prompt or a CLI command, the NuGet key must be provided through the NuGet.Config file. 
+
+You have the option to create a NuGet.Config file on the application level (it will overwrite any global NuGet.Config files and configuration will be applicable only for the specific application) or to use a global NuGet.Config file (it will be valid for all applications unless explicitly overwritten by an application NuGet.Config file). [Learn more details about NuGet.Config configuration and location on each operating system here...](https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior)
+
+
+- Close all instances of Visual Studio.
+
+- Navigate to the directory where the NuGet.Config file resides. [Learn where is the location of NuGet.Config on each operating system here...](https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior#config-file-locations-and-uses)
+
+- Open the NuGet.Config file or create one if the file does not exist.
+
+- Add the Telerik NuGet server in the package source section through a custom key (in the demo case, the key is named **MyTelerikFeed**)
+
+- Add the generated Telerik NuGet key through the **packageSourceCredentials** section by adding a **Username** key with the value **api-key** and a **ClearTextPassword** key with the value of the generated Telerik NuGet key.
+
+    An example of adding the Telerik NuGet server with an API key.
```

Comment:
As the previous list items explain the content of the code snippet, you can remove this introductory sentence.